Userland/Games/GameOfLife/main.cpp

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2021, Andres Crucitti <dasc495@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+#include "BoardWidget.h"
+#include <Games/GameOfLife/GameOfLifeGML.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/BoxLayout.h>
+#include <LibGUI/Button.h>
+#include <LibGUI/Icon.h>
+#include <LibGUI/Label.h>
+#include <LibGUI/Menu.h>
+#include <LibGUI/Menubar.h>
+#include <LibGUI/MessageBox.h>
+#include <LibGUI/Slider.h>
+#include <LibGUI/SpinBox.h>
+#include <LibGUI/Statusbar.h>
+#include <LibGUI/Toolbar.h>
+#include <LibGUI/ToolbarContainer.h>
+#include <LibGUI/Window.h>
+
+const char* click_tip = "Tip: click the board to toggle individual cells, or click+drag to toggle multiple cells";
+
+int main(int argc, char** argv)
+{
+
+    auto app = GUI::Application::construct(argc, argv);
+    auto app_icon = GUI::Icon::default_icon("app-gameoflife");
```

Comment:
You don't seem to have an icon for this. :^) Don't worry though, you can add one later.

Reply:
If this is replacing the `Conway` application then the `conway` icons could be renamed to `gameoflife` and used here.